src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java

```diff
@@ -193,4 +223,19 @@ public SearchReadsRequest apply(Contig shard) {
                   contigs.iterator().next())));
     }
   }
+
+  private static Iterable<Contig> parseContigs(String c) {
```

Comment:
This method can go away.  There is a version of it in utils-java.

See https://github.com/googlegenomics/dataflow-java/blob/master/src/main/java/com/google/cloud/genomics/dataflow/utils/GenomicsDatasetOptions.java#L50